app/app.go

```diff
@@ -679,6 +679,11 @@ func (a *App) GetT() goi18n.TranslateFunc {
 	return a.t
 }
 
+
+func (a *App) CheckIntegrity() <-chan model.IntegrityCheckResult {
```

Comment:
Hi. Thanks for the contribution.

This change seems incorrect. you are just returning an empty channel.
instead, it should forward the work to the Store method.

Also, please run `gofmt -s -w` on the source code before committing to conform to code style.